easybuild/easyconfigs/l/leidenalg/leidenalg-0.10.1-foss-2022b.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'leidenalg'
+version = '0.10.1'
+
+homepage = 'https://github.com/vtraag/leidenalg'
+description = """Implementation of the Leiden algorithm for various quality
+functions to be used with igraph in Python."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+    ('pkgconf', '1.9.3'),
+    ('PyYAML', '6.0'),
+    ('Bison', '3.8.2'),
+    ('libtool', '2.4.7'),
+    ('flex', '2.6.4'),
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('igraph', '0.10.6'),
+    ('python-igraph', '0.10.6'),
+    ('libleidenalg', '0.11.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
```

Comment:
I see there was a 

```
local_preinstallopts = "python setup.py build --use-pkg-config && "
```

In leidenalg-0.9.1-foss-2022a.eb. Can I assume that you removed that intentionally and that it is not needed anymore? (I'm not sure why it was there, there is no comment to indicate why it was needed, so it is hard to say if it still is)

Reply:
I do not know why "--use-pkg-config" was in the old. Seems to build without it.

Reply:
Ok, let's go without it. If it was there for a reason, I'm sure we'll hit it again - and then hopefully document properly what it fixed :P